action-server/covidflow/actions/test_navigation_form.py

```diff
@@ -304,4 +309,44 @@ def _generate_description(
     ):
         description += f" {description_parts.get('contact_before_visit', '')}"
 
+    opening_hours_part: Dict[str, str] = description_parts["opening_hours"]
+    if len(location.opening_hours) > 0:
+        description += f"\n\n{_format_opening_hours(location.opening_hours, language, opening_hours_part)}"
+
     return description
+
+
+def _format_time(time: time, language: str):
+    return (
+        time.strftime(SHORT_TIME_FORMAT[language])
+        if time.minute == 0
+        else time.strftime(LONG_TIME_FORMAT[language])
+    ).lower()
+
+
+def _format_hours(
```

Comment:
Maybe `_format_opening_intervals`, since the distinction between `_format_hours` and `_format_opening_hours` is unclear